docs/requirements.txt

```diff
@@ -10,4 +10,5 @@ sphinx-autoapi
 sphinx-copybutton
 sphinx-book-theme
 sphinx-design
-git+https://github.com/astronomy-commons/hipscat.git@main
+git+https://github.com/astronomy-commons/hipscat.git@hats
```

Comment:
Noting that the name of the repo itself still has "hipscat" in it.  I assume that's a change we're obliged to make in a separate PR, but I'm just wanting to note it.

Reply:
Yes - we need to wait until the repo has actually been renamed, which should be relatively soon.